File: internal/collector/treesitter/testdata/test_kind/java/java-testng/setup/SetupTest.java
// SPDX-License-Identifier: Apache-2.0

package fixture;

import org.testng.annotations.BeforeClass;
import org.testng.annotations.BeforeMethod;

public class SetupTest {
    @BeforeClass
    public void setUpAll() {}

    @BeforeMethod
    public void setUp() {}
}

File: internal/collector/treesitter/testdata/test_kind/scala/scala-scalatest/setup/SetupTest.scala
// SPDX-License-Identifier: Apache-2.0

package fixture

import org.scalatest.funsuite.AnyFunSuite

// ScalaTest setup uses call-expression beforeAll/beforeEach forms; the Bucket
// B test_block predicate (chunker_scala.go classifyTestBlockScala) classifies
// these calls as TestKindSetup.
class SetupTest extends AnyFunSuite {
  beforeAll {
    val _ = 1
  }
  beforeEach {
    val _ = 2
  }
}
